samples/dashboards/demo/minimal/demo.html

```diff
@@ -5,10 +5,12 @@
 <script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
 
 <div id="container"></div>
-
+<p class="highcharts-description">
+  The basic dashboards showing Vitamin A and Iron amount in foods.
```

Comment:
You can describe that the data is shown on the chart and also in the table. You can mention the recommended daily dose. It should be more visible on the new designs.

Reply:
- We mention what kind of data is, but I'm not pretty sure about the source of data
- I would avoid recommendation something in our caption